pages/index.js

```diff
@@ -208,100 +238,55 @@ function getFontColor({ value, conflict, prefilled }) {
   return false;
 }
 
+function getThemeColors(theme = 'Green') {
+  return ThemeColors[theme] || [];
+}
+
 class GenerationUI extends Component {
   constructor(props) {
     super(props);
 
     this.handler = this.handler.bind(this);
 
 
-    this.state = { value: 30 };
-    this
+    this.state = {
+      theme: '',
+      value: 30,
+    };
   }
 
   generateGame = () => {
-    console.log('hello')
-    console.log('this is value: ',this.state.value)
     this.props.generateGame(this.state.value);
-    this.forceUpdate()
+    this.forceUpdate();
   }
 
   handler = (e) => {
-    console.log('second e: ', e.target.id);
-    let target = e.target.id;
-    let color1;
-    let color2;
-    let color3;
-
-    if (target == 'green') {
-        color1 = 'rgb(0,150,0)';
-        color2 = 'rgb(0,100,0)';
-        color3 = 'rgb(0,255,0)';
-    }
-    if (target == 'blue') {
-        color1 = 'rgb(0,0,150)';
-        color2 = 'rgb(155,155,255)';
-        color3 = 'rgb(0,0,255)';
-    }
-    if (target == 'purple') {
-        color1 = 'rgb(150,0,150)';
-        color2 = 'rgb(255,0,255)';
-        color3 = 'rgb(100,0,100)';
-    }
-    if (target == 'red') {
-        color1 = 'rgb(175,0,0)';
-        color2 = 'rgb(150,0,0)';
-        color3 = 'rgb(255,0,0)';
-    }
-    if (target == 'orange') {
-        color1 = 'rgb(255,175,0)';
-        color2 = 'rgb(255,140,0)';
-        color3 = 'rgb(255,100,0)';
-    }
-    if (target == 'yellow') {
-        color1 = 'rgb(225,225,0)';
-        color2 = 'rgb(255,255,0)';
-        color3 = 'rgb(200,200,0)';
-    }
+    const target = e.target.id;
+    const colors = getThemeColors(target);
 
+    colors.forEach((color, index) => {
```

Comment:
We've changed the theme colors to an array, so we can use array methods like `forEach`. I would recommend using a theme manager or create a theme context instead of mutating CSS variables